src/components/Avatar/index.js

```diff
@@ -1,22 +1,37 @@
-import React from "react"
-import PropTypes from "prop-types"
-import { cx } from "emotion"
-import { imgWrapper, wrapper } from "./styles"
+import React from "react";
+import PropTypes from "prop-types";
+import { cx } from "emotion";
+import { imgWrapper, wrapper } from "./styles";
+import { UserType } from '../../type';
 
 Avatar.propTypes = {
   className: PropTypes.string,
   size: PropTypes.oneOf(["medium", "large"]),
-  src: PropTypes.string
+  src: UserType.image.PropTypes.string
 }
 
-function Avatar({ size = "medium", src, className }) {
-  return (
-    <div className={cx(wrapper, { large: size === "large" }, className)}>
-      <div className={imgWrapper}>
-        <img src={src} alt="avatar" />
+if ( src !== '' ) {
+  const Avatar = ({ size = "medium", src, className }) => {
```

Comment:
Avatar should remain a function that receives some props. The conditional rendering should happen inside that function.